gqcpy/include/interfaces.hpp

```diff
@@ -699,4 +700,145 @@ void bindSimple2DMInterface(Class& py_class) {
 }
 
 
+/*
+ *  MARK: QCModel - Hartree-Fock
+ */
+
+/**
+ *  Add Python bindings for some APIs related to `QCModel::R/U/GHF`.
+ * 
+ *  @tparam Class               The type of the Pybind11 `class_` (generated by the compiler).
+ *  
+ *  @param py_class             The Pybind11 `class_` that should obtain APIs related to `QCModel::R/U/GHF`.
+ */
+template <typename Class>
+void bindQCModelHartreeFockInterface(Class& py_class) {
+
+    // The C++ type corresponding to the Python class.
+    using Type = typename Class::type;
+    using Hamiltonian = typename Type::Hamiltonian;
+    using Scalar = typename Type::Scalar;
+
+    py_class
+        .def(
+            "calculateOrthonormalBasis1DM",
+            [](const Type& parameters) {
+                return parameters.calculateOrthonormalBasis1DM();
+            },
+            "Return the 1-DM expressed in an orthonormal basis related to these optimal HF parameters.")
+
+        .def(
+            "calculateScalarBasis1DM",
+            [](const Type& parameters) {
+                return parameters.calculateScalarBasis1DM();
+            },
```

Comment:
```suggestion
            &Type::calculateScalarBasis1DM,
```

Reply:
The issue here is analogous to the `orthonormalBasis1DM` binding.